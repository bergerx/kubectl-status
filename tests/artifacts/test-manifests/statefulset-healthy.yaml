# wait-for: jsonpath={.status.replicas}=3
# wait-for: jsonpath={.status.readyReplicas}=3
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: web
spec:
  replicas: 3
  selector:
    matchLabels:
      app: nginx
  serviceName: nginx
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - image: k8s.gcr.io/nginx-slim:0.8
        name: nginx
        ports:
        - containerPort: 80
          name: web
          protocol: TCP
        volumeMounts:
        - mountPath: /usr/share/nginx/html
          name: www
  volumeClaimTemplates:
  - apiVersion: v1
    kind: PersistentVolumeClaim
    metadata:
      name: www
    spec:
      accessModes:
      - ReadWriteOnce
      resources:
        requests:
          storage: 1Mi
      volumeMode: Filesystem
