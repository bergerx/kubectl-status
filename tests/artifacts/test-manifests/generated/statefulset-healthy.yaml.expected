
Printed all existing resource statuses, starting to watch. Switching to shallow mode during watch!


StatefulSet/web -n statefulset-healthy, created Xs ago, gen:1
  InProgress: Replicas: 0/3
    Reconciling: LessReplicas, Replicas: 0/3
  desired:3, ready:0, current:0
  Outage: StatefulSet has no Ready replicas.
  Ongoing rollout: Waiting for statefulset spec update to be observed...
  Stuck Rollout?: Still replacing the first Pod, may indicate a stuck rollout.
  Known/recorded manage events:
    Xs ago Updated by kubectl-client-side-apply (metadata, spec)

StatefulSet/web -n statefulset-healthy, created Xs ago, gen:1
  InProgress: Replicas: 1/3
    Reconciling: LessReplicas, Replicas: 1/3
  desired:3, existing:1, ready:0, current:1, updated:1
  Outage: StatefulSet has no Ready replicas.
  Stuck Initial Rollout? First rollout not yet progressed.
  Ongoing rollout: Waiting for 3 pods to be ready...
  Stuck Rollout?: Still replacing the first Pod, may indicate a stuck rollout.
  Known/recorded manage events:
    Xs ago Updated by kube-controller-manager (status)
    Xs ago Updated by kubectl-client-side-apply (metadata, spec)

StatefulSet/web -n statefulset-healthy, created Xs ago, gen:1
  InProgress: Replicas: 2/3
    Reconciling: LessReplicas, Replicas: 2/3
  desired:3, existing:2, ready:1, current:2, updated:2
  Ongoing rollout: Waiting for 2 pods to be ready...
  Stuck Rollout?: Still replacing the first Pod, may indicate a stuck rollout.
  Known/recorded manage events:
    Xs ago Updated by kubectl-client-side-apply (metadata, spec)
    Xs ago Updated by kube-controller-manager (status)

StatefulSet/web -n statefulset-healthy, created Xs ago, gen:1
  InProgress: Ready: 2/3
    Reconciling: LessReady, Ready: 2/3
  desired:3, existing:3, ready:2, current:3, updated:3
  Ongoing rollout: Waiting for 1 pods to be ready...
  Stuck Rollout?: Still replacing the first Pod, may indicate a stuck rollout.
  Known/recorded manage events:
    Xs ago Updated by kubectl-client-side-apply (metadata, spec)
    Xs ago Updated by kube-controller-manager (status)

StatefulSet/web -n statefulset-healthy, created Xs ago, gen:1
  Current: Partition rollout complete. updated: 3
  desired:3, existing:3, ready:3, current:3, updated:3
  Known/recorded manage events:
    Xs ago Updated by kubectl-client-side-apply (metadata, spec)
    Xs ago Updated by kube-controller-manager (status)

StatefulSet/web -n statefulset-healthy, created Xs ago, gen:1
  Current: Partition rollout complete. updated: 3
  desired:3, existing:3, ready:3, current:3, updated:3
  Rollouts: Use the `--include-rollout-diffs` flag to see the diff between each rollout!
    Xs ago used ControllerRevision/web-b46f789c4.
  Known/recorded manage events:
    Xs ago Updated by kubectl-client-side-apply (metadata, spec)
    Xs ago Updated by kube-controller-manager (status)
  Events:
    SuccessfulCreate Xs ago from statefulset-controller: create Claim www-web-0 Pod web-0 in StatefulSet web success
    SuccessfulCreate Xs ago from statefulset-controller: create Pod web-0 in StatefulSet web successful
    SuccessfulCreate Xs ago from statefulset-controller: create Claim www-web-1 Pod web-1 in StatefulSet web success
    SuccessfulCreate Xs ago from statefulset-controller: create Pod web-1 in StatefulSet web successful
    SuccessfulCreate Xs ago from statefulset-controller: create Claim www-web-2 Pod web-2 in StatefulSet web success
    SuccessfulCreate Xs ago from statefulset-controller: create Pod web-2 in StatefulSet web successful

StatefulSet/web -n statefulset-healthy, created Xs ago, gen:1
  Current: Partition rollout complete. updated: 3
  desired:3, existing:3, ready:3, current:3, updated:3
  Known/recorded manage events:
    Xs ago Updated by kubectl-client-side-apply (metadata, spec)
    Xs ago Updated by kube-controller-manager (status)
